### DISCUSSION TITLE
Accent Colors

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#334] [General]
discussion

**Comment by redstone-dev** [comment]
I think it would be cool to have a set of pre-picked ones and the ability to have one custom one.

**Reply by jdev082** [reply]
A prerequisite is to switch to normal CSS.

**Reply by redstone-dev** [reply]
No? Just use JavaScript?

**Reply by jdev082** [reply]
@redstone-scratch Catalyst 5.0 won't have tailwind

**Reply by redstone-dev** [reply]
You. Do. Not. Need. Tailwind. For. This.

**Reply by jdev082** [reply]
Catalyst uses tailwind. That's an issue

**Reply by redstone-dev** [reply]
> Catalyst uses tailwind. That's an issue

Ok but you don't need tailwind??? just make a class which some javascript selects and adds the appropriate tailwind styles to??

**Comment by jdev082** [comment]
#104

**Comment by jdev082** [comment]
This feature is on hold until v3.5 is complete

**Comment by jdev082** [comment]
Will be implemented in v4.0.0
